frontend/apps/docs/content/docs/url-based-access.mdx

```diff
@@ -3,3 +3,24 @@ title: URL-Based Access
 ---
 
 TBD
+
+## Detect schema format
+
+You can instantly generate an ER diagram by appending a GitHub file URL to `liambx.com/erd/p/`. For example:
+
+https://liambx.com/erd/p/github.com/mastodon/mastodon/blob/main/db/schema.rb
```

Comment:
This URL is not working now because is is not raw content.
It would be better to correspond to CLI specification? (`--input`)

Reply:
I want to keep it this way, because we want to provide an experience for users that they can see the ER diagram by adding a string in front of URL!